setup.py

```diff
@@ -23,16 +23,15 @@ def tuple_list(directory):
     README = ''
 
 # Copy all data_files from the 'static'- and 'templates'-folder into their destination
-data_files_list = tuple_list('static')
-data_files_list.extend(tuple_list('templates'))
-# TODO: also add config-file to data_files_list
+data_files_list = tuple_list('flask-dashboard/static')
+data_files_list.extend(tuple_list('flask-dashboard/templates'))
```

Comment:
When creating the setup script, I also looked into the manifest.in file, but I couldn't figure out how to declare the static files. This function contains the same functionality